homeassistant/components/guardian/switch.py

```diff
@@ -91,7 +91,11 @@ def __init__(
     ) -> None:
         """Initialize."""
         super().__init__(
-            entry, coordinators, "valve", "Valve Controller", None, "mdi:water"
+            entry,
+            coordinators,
+            SwitchEntityDescription(
+                key="valve", name="Valve Controller", icon="mdi:water"
+            ),
```

Comment:
Even if it's only one description, I would recommend to define it as constant instead.